styleguide/components/oisc-warnings/oisc-warning.stories.mdx

```diff
@@ -0,0 +1,86 @@
+import { Meta, Story, Preview, Source } from '@storybook/addon-docs/blocks';
+import initOISCWarnings from '../../../src/js/oisc-warnings';
+
+<Meta
+  title="Components/OISC Warning"
+  component={() => templateDefault}
+  decorators={[
+    (Story) => {
+      setTimeout(() => initOISCWarnings());
+      return `<div style="max-width: 800px">${Story()}</div>`;
+    },
+  ]}
+/>
+
+# The Office of the Immigration Services Commissioner (OISC) warnings
+
+OISC warnings show advisers what regulations apply to specific content.
+
+- Content designers can input free text into the heading fields
+- Content designers can input free text and links into the paragraph fields
+- Adding content to the paragraph is optional
+
+## Variant: With title and description
+
+- Content designers can input free text into the heading fields
+- Content designers can input free text and links into the paragraph fields
+
+import templateDefault from './_default.html.haml';
+import templateDefaultCode from '!!raw-loader!./_default.html.haml';
+
+<Preview>
+  <Story
+    name="With title and description"
+    parameters={{ docs: { source: { code: templateDefaultCode } } }}
```

Comment:
This little dance sets the code snippet. The default behaviour is to return the template string which resolves to a literal `() => templateDefault`. Wish this syntax was simpler but think It's worth it to be able to show good code-snippets.